app/helpers/application_helper.rb

```diff
@@ -11,111 +11,103 @@ def login_helper(style = '')
   end
 
   def side_panel_items
-    [
-      {
-        url: root_path,
-        title: 'Home'
-      },
-      {
-        url: 'path-to-be-added',
-        title: 'Team hub'
-      },
-      {
-        url: 'path-to-be-added',
-        title: 'Lifestyle'
-      },
-      {
-        url: 'path-to-be-added',
-        title: 'Dealbook'
-      },
-      {
-        url: 'path-to-be-added',
-        title: 'Video'
+    pages = []
+    pages << {
+      url: @is_admin_panel ? admin_root_path : root_path,
+      title: 'Home' }
+
+    static_pages = Category.all
+                           .reject { |page| page.category_type == 'articles' }
+                           .sort_by{ |page| page.position}
```

Comment:
I think it could be part of query to DB, so there will be no need use ruby to filter, sort